R/crmPack-package.R

```diff
@@ -28,14 +28,12 @@
 #  cpp_coxfit
 #' @import checkmate
 #' @import ggplot2
+#' @import methods 
 #' @importFrom gridExtra arrangeGrob
 #' @importFrom graphics plot hist legend lines matlines matplot
-#' @importFrom methods setClass setOldClass setGeneric setMethod representation
-#' signature prototype initialize new is .valueClassTest as callNextMethod slot 
-#' slotNames `slot<-` show formalArgs validObject
 #' @importFrom stats binomial coef cov2cor gaussian glm lm median model.matrix
 #' optim pgamma plogis pnorm qgamma qlogis qnorm quantile rbinom rgamma approxfun
-#' rnorm runif uniroot var vcov
+#' rnorm runif uniroot var vcov step
```

Comment:
no need to update this list. during clean up we prefix all stats function calls with `stats::`. therefore at the end we will reduce this list of imported functions to a single function only (only for R CMD check happiness)